docker/Dockerfile.focal

```
@@ -13,8 +13,10 @@ deb-src http://apt.llvm.org/focal/ llvm-toolchain-focal-${LLVM_VERSION} main\n"
     curl -L https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4052245BD4284CDD
 
-RUN curl -L --output /tmp/cmake.tar.gz \
-  https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-linux-x86_64.tar.gz \
+ARG ARCH
+RUN ARCH=$(uname -m) \
+  && curl -L --output /tmp/cmake.tar.gz \
+  https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-linux-${ARCH}.tar.gz \
```

I don't think that this is going to do what you intend. The assignment to `ARCH` in `RUN` will override whatever was set by user, so you may as well do
```suggestion

RUN curl -L --output /tmp/cmake.tar.gz \
  https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-linux-$(uname -m).tar.gz \
```

---

I was experimenting changes to get eBPF working on Yocto, and bpftrace is a tool I could use to verify functionalities on target system. So I had to build the project and hit compile errors.